autoinstall/FreeBSD/installerconfig

```diff
@@ -0,0 +1,140 @@
+
+debug=yes
+nonInteractive=yes
+
+{% if boot_disk_controller == 'nvme' %}
+PARTITIONS="nvd0 gpt"
+{% elif boot_disk_controller == 'sata' or boot_disk_controller == 'ide' %}
+PARTITIONS="ada0 gpt"
+{% else %}
+PARTITIONS="da0 gpt"
+{% endif %}
+
+{% if guest_id.find('64Guest') != -1 %}
+DISTRIBUTIONS="kernel.txz base.txz kernel-dbg.txz lib32.txz src.txz ports.txz"
+{% else %}
+DISTRIBUTIONS="kernel.txz base.txz kernel-dbg.txz src.txz ports.txz"
+{% endif %}
+
+#!/bin/sh
+# Set hostname
+sysrc hostname="FreeBSD-{{ hostname_timestamp }}"
+
+{% if boot_disk_controller == 'nvme' %}
+gpart bootcode -b /boot/pmbr -p /boot/gptboot -i 1 nvd0
+{% elif boot_disk_controller == 'sata' or boot_disk_controller == 'ide' %}
+gpart bootcode -b /boot/pmbr -p /boot/gptboot -i 1 ada0
+{% else %}
+gpart bootcode -b /boot/pmbr -p /boot/gptboot -i 1 da0
+{% endif %}
+
+# Set Time Zone to UTC
+echo "Setting Time Zone to UTC ..."
+/bin/cp /usr/share/zoneinfo/UTC /etc/localtime
+/usr/bin/touch /etc/wall_cmos_clock
+/sbin/adjkerntz -a
+
+echo "Set network interface with DHCP IP assignment ..." > /dev/ttyu0
+ifdev=$(ifconfig | grep '^[a-z]' | cut -d: -f1 | head -n 1)
+echo "Get ifname ${ifdev}" > /dev/ttyu0
+sysrc ifconfig_${ifdev}=DHCP
+
+# Get DHCP for nic0
+echo "Get IP with dhclient ..." > /dev/ttyu0
+dhclient ${ifdev}
+sleep 15
+echo "Check network ..." > /dev/ttyu0
+ifconfig > /dev/ttyu0
+
+# Set Proxy. without the proxy, maybe it will take more time to download pakages xorg & gnome
```

Comment:
Please update this comment as you didn't download xorg & gnome from online repo

Reply:
Done. Thanks